newsapi_headlines-to-email/main.bal

```diff
@@ -0,0 +1,30 @@
+import ballerina/log;
+import ballerinax/newsapi;
+import wso2/choreo.sendemail;
```

Comment:
Why using choreo?

Reply:
Choreo sendmail was used for the ease of testing these examples for users. Compared to the ballerina email connector where a host need to be setup separately, choreo.sendmail can be used to send the email without any additional effort.

Reply:
Updated in [5131705](https://github.com/ballerina-guides/integration-samples/pull/35/commits/51317055927a041c645786b20bcb6af3864f1081)